projects/ngx-wig/src/lib/config.ts

```diff
@@ -59,4 +62,18 @@ export const DEFAULT_LIBRARY_BUTTONS: TButtonLibrary = {
   }
 };
 
+export const CUSTOM_LIBRARY_BUTTONS: TButtonLibrary = {
```

Comment:
I really like the simplicity of this approach. Definitely much cleaner than previous suggestions. I have some concerns though.

This approach seems a good fit for button plugins. So, I would suggest to make it clear to the consumers of the library that they can create **only** button plugins and not generally plugins (that would involve interaction with their components and we would need, possibly, more than a simple command handler) 😉

Reply:
I think with buttons we can cover 9 of 10 cases.

Reply:
Should we name it `ButtonPlugin` or `CustomButton` or something that denotes button so that it is more clear?